docker file update

tyu-avo opened the pull request:
For first time container creation with an empty database, the postCreateCommand.sh file was failing due to the container name difference. 

This may be a naming convention difference from using docker desktop.

![containerName](https://user-images.githubusercontent.com/87871433/190238719-b908f67d-015f-4e98-84ed-e23240bc7347.PNG)

tyu-avo commented:
@WalterMoar to review behavior went available  :)

WalterMoar commented:
> @WalterMoar to review behavior went available :)

I deleted all my containers and volumes and did a rebuild. The PostgreSQL container still comes out as _queue-management_devcontainer_db_1_, so it looks like this _postCreateCommand.sh_ change will break things for anyone using _dockerd_ in WSL2.

Hopefully Docker Desktop respects the _container_name_ property on the container. Can you try this change instead, and see what name it gives your container?

![image](https://user-images.githubusercontent.com/35532993/190278932-297b5d0b-4530-4103-8655-2a611d0190ce.png)

tyu-avo commented:
> > @WalterMoar to review behavior went available :)
> 
> I deleted all my containers and volumes and did a rebuild. The PostgreSQL container still comes out as _queue-management_devcontainer_db_1_, so it looks like this _postCreateCommand.sh_ change will break things for anyone using _dockerd_ in WSL2.
> 
> Hopefully Docker Desktop respects the _container_name_ property on the container. Can you try this change instead, and see what name it gives your container?
> 
> ![image](https://user-images.githubusercontent.com/35532993/190278932-297b5d0b-4530-4103-8655-2a611d0190ce.png)

It works! I've updated the PR accordingly.
Thanks!

WalterMoar commented:
Lesson learned - always set the name of the container, if you need to reference it elsewhere. Thank you!

WalterMoar commented:
@josekudiyirippil - this can be merged whenever you have time. It doesn't need testing or a deployment as it only affects the containers in VSCode.
